### ISSUE TITLE
Trying to make a pickit line for a specialized ring

### ISSUE THREAD

**Issue by MrBeatDown** [kolton/d2bot-with-kolbot#2162]
Im trying to collect rare rings with the    5% Bonus to Attack Rating
But i cannot figure out how to add it to the line.  Nipchecker always fails to pass any line combination i try with it. Does anyone know if its possible to pick these rings. I want ALL rings with this trait and i will decided which one to trash after testing. Thanks.

**Comment by InterestingIndeed** [comment]
What is the pickit you have so far for that ring? It is always good to provide some example.

**Comment by MrBeatDown** [comment]
Yeah that works. I cant believe i couldnt figure that out.\..... Thanks alot./
